bot/cogs/guess_number.py

```diff
@@ -0,0 +1,100 @@
+import discord
+import math, random
+from discord.ext.commands import Bot, Cog, command
+
+
+class guess_game(Cog): #(Cog): # Include Cog if it's a part of Cog file in Friendo Bot. Change class name if needed.
+    """Commands for Guessing Right Number"""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    # Variables in class guess_game
+    #Default bounds, random number, and rounds.
+    lower_bound = 1
```

Comment:
these class variables could be added above the `__init__`, I find that to be easier to read.